frontend/app/resources/power-platform/client-friendly-statuses.json

```
@@ -3,62 +3,62 @@
       "esdc_clientfriendlystatusid": "51af5170-614e-ee11-be6f-000d3a09d640"
     },
     {
       "@odata.etag": "W/\"68387614\"",
-      "esdc_descriptionfrench": "Nous avons examiné la demande de votre enfant au Régime canadien de soins dentaires. \n\nNos dossiers montrent qu'une demande a déjà été soumise pour l'enfant. Cette demande en double a été annulée. \n\nVeuillez contacter Service Canada en appelant au 1-833-537-4342 pour plus d'informations. Pour les services ATS (téléscripteur), veuillez composer le 1-833-677-6262. Vous pouvez également vous rendre dans un Centre Service Canada.",
-      "esdc_descriptionenglish": "We have reviewed your child's application for the Canadian Dental Care Plan. \n\nOur records show that an application has already been submitted for the child. As a result, this application has been cancelled.\n\nFor more information, contact Service Canada by calling 1-833-537-4342. For TTY (teletypewriter) services call 1-833-677-6262. You can also visit a Service Canada Centre.",
+      "esdc_descriptionfrench": "Nous avons examiné la demande de votre enfant au Régime canadien de soins dentaires. \n\nNos dossiers montrent qu'une demande a déjà été soumise pour l'enfant. Cette demande en double a été annulée. \n\nVeuillez contacter Service Canada en appelant au 1-833-537-4342 pour plus d'informations. Pour les services ATS (téléscripteur), veuillez composer le 1-833-677-6262. Vous pouvez également vous rendre dans un Centre Service Canada.",
+      "esdc_descriptionenglish": "We have reviewed your child’s application for the Canadian Dental Care Plan. \n\nOur records show that an application has already been submitted for the child. As a result, this application has been cancelled.\n\nFor more information, contact Service Canada by calling 1-833-537-4342. For TTY (teletypewriter) services call 1-833-677-6262. You can also visit a Service Canada Centre.",
```

Should we change these apostrophes to be consistent with the rest of the application like https://github.com/DTS-STN/canadian-dental-care-plan/pull/1926?

```suggestion
      "esdc_descriptionenglish": "We have reviewed your child's application for the Canadian Dental Care Plan. \n\nOur records show that an application has already been submitted for the child. As a result, this application has been cancelled.\n\nFor more information, contact Service Canada by calling 1-833-537-4342. For TTY (teletypewriter) services call 1-833-677-6262. You can also visit a Service Canada Centre.",
```

I noticed the above PR also indicates changes to spaces and hyphens.

(cc @Fbasham)

---

yeah, should be changed here as well.

---

I agree because we have control over this dataset, but disagree since we wouldn't have this control if we were calling their API directly. 🤷

---

we've already set a precedent in the repo.  No need to go against that now until we're pulling real data from PP.

---

Fair enough

---

Fixed apostrophes